package-lock.json

```diff
@@ -6464,7 +6464,8 @@
       "version": "1.5.1",
```

Comment:
Since package.json didn't change, package-lock.json shouldn't change either. Could you double check your NPM version (should be 18.18 as Austin said), then reset changes to package-lock.json, then run npm install again to see if you still see package-lock.json change?

Reply:
THat's may be the reason.  I upgraded to node 18.18.2 and npm to 9.8.1.